src/OrbitQt/orbithistogram.h

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_HISTOGRAM_H_
+#define ORBIT_HISTOGRAM_H_
+
+#include <qevent.h>
+#include <qpainter.h>
+#include <qwidget.h>
```

Comment:
Please use the class-name includes for Qt-stuff, i.e. `#include <QEvent>` and so on.